pkg/ingester/errors_test.go

```diff
@@ -348,151 +348,305 @@ func TestWrapOrAnnotateWithUser(t *testing.T) {
-			err:  fmt.Errorf("wrapped: %w", newTSDBUnavailableError("tsdb stopping")),
+		"a wrapped tsdbUnavailableError gets translated into an errorWithHTTPStatus 503 error": {
+			err: fmt.Errorf("wrapped: %w", newTSDBUnavailableError("tsdb stopping")),
 			expectedTranslation: newErrorWithHTTPStatus(
 				fmt.Errorf("wrapped: %w", newTSDBUnavailableError("tsdb stopping")),
 				http.StatusServiceUnavailable,
 			),
 		},
-		{
-			name: "a sampleError gets translated into an errorWithHTTPStatus 400 error",
-			err:  newSampleError("id", "sample error", timestamp, labelAdapters),
+		"a sampleError gets translated into an errorWithHTTPStatus 400 error": {
+			err: newSampleError("id", "sample error", timestamp, labelAdapters),
 			expectedTranslation: newErrorWithHTTPStatus(
 				newSampleError("id", "sample error", timestamp, labelAdapters),
 				http.StatusBadRequest,
 			),
 		},
-		{
-			name: "a wrapped exemplarError gets translated into an errorWithHTTPStatus 400 error",
-			err:  fmt.Errorf("wrapped: %w", newSampleError("id", "sample error", timestamp, labelAdapters)),
+		"a wrapped sample gets translated into an errorWithHTTPStatus 400 error": {
+			err: fmt.Errorf("wrapped: %w", newSampleError("id", "sample error", timestamp, labelAdapters)),
 			expectedTranslation: newErrorWithHTTPStatus(
 				fmt.Errorf("wrapped: %w", newSampleError("id", "sample error", timestamp, labelAdapters)),
 				http.StatusBadRequest,
 			),
 		},
-		{
-			name: "a exemplarError gets translated into an errorWithHTTPStatus 400 error",
-			err:  newExemplarError("id", "exemplar error", timestamp, labelAdapters, labelAdapters),
+		"a exemplarError gets translated into an errorWithHTTPStatus 400 error": {
+			err: newExemplarError("id", "exemplar error", timestamp, labelAdapters, labelAdapters),
 			expectedTranslation: newErrorWithHTTPStatus(
 				newExemplarError("id", "exemplar error", timestamp, labelAdapters, labelAdapters),
 				http.StatusBadRequest,
 			),
 		},
-		{
-			name: "a wrapped exemplarError gets translated into an errorWithHTTPStatus 400 error",
-			err:  fmt.Errorf("wrapped: %w", newExemplarError("id", "exemplar error", timestamp, labelAdapters, labelAdapters)),
+		"a wrapped exemplarError gets translated into an errorWithHTTPStatus 400 error": {
+			err: fmt.Errorf("wrapped: %w", newExemplarError("id", "exemplar error", timestamp, labelAdapters, labelAdapters)),
 			expectedTranslation: newErrorWithHTTPStatus(
 				fmt.Errorf("wrapped: %w", newExemplarError("id", "exemplar error", timestamp, labelAdapters, labelAdapters)),
 				http.StatusBadRequest,
 			),
 		},
-		{
-			name: "a perUserMetadataLimitReachedError gets translated into an errorWithHTTPStatus 400 error",
-			err:  newPerUserSeriesLimitReachedError(10),
+		"a perUserMetadataLimitReachedError gets translated into an errorWithHTTPStatus 400 error": {
+			err: newPerUserSeriesLimitReachedError(10),
```

Comment:
The error description talks about per-user-metadata-limit but the instantiated error is per-user-series-limit. Same for the next three

Reply:
Thank you for this, I've fixed it.